handlers/search.go

```diff
@@ -135,11 +135,14 @@ func SearchAhead(w http.ResponseWriter, r *http.Request) {
 				http.Error(w, "Internal server error", http.StatusInternalServerError)
 				return
 			}
-			err = db.ReaderDb.Select(result, `
-				SELECT ENCODE(txhash::bytea, 'hex') AS txhash
-				FROM blocks_transactions
-				WHERE txhash = $1
-				ORDER BY block_slot LIMIT 10`, txHash)
+			tx, dbErr := db.BigtableClient.GetIndexedEth1Transaction(txHash)
```

Comment:
If you follow my comment for https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2051/files#diff-143a07e5aa31442d87e5dd9584d1525fc98e8b2fef161a08af276d65f60184acR1785, you will not need `dbErr` here. You could use `err` directly.

Reply:
ok